HealthcareSystem/PatientWebApp/Controllers/PatientController.cs

```diff
@@ -71,5 +71,25 @@ public ActionResult AddPatient(PatientDTO patientDTO)
             }
             return Ok();
         }
+
+        /// <summary>
+        /// /activate patient (property: IsActive) to true
+        /// </summary>
+        /// <param name="id">id of the object to be changed</param>
+        /// <returns>if alright returns code 200(Ok), if not 400(bed request)</returns>
+        [HttpPut("{jmbg}")]
+        public ActionResult ActivatePatient(string jmbg)
```

Comment:
So this on the other hand definitely does need to be an endpoint, but looking at it, does the path 'api/mail/{jmbg}' look like something you would use for account activation? Wouldn't something like 'api/user/activate/{id}' or 'api/patients/activate/{id}' make more sense? Also, I assume you know this and that it isn't implemented because it isn't exactly our concern right now, but it should really be {horrible encrypted string} instead of {jmbg}.

Reply:
Correction: the initial path was 'api/patients/{jmbg}', not 'api/mail/{jmbg}' (although the point still stands). Changed to 'api/patients/activate/{jmbg}' in f3dd33a7ec46560896650fb783d63778066c2722.